packages/block-library/src/blocks/supernova/editor-styles.scss

```diff
@@ -1,13 +1,40 @@
 .supernova {
   
   > .wp-block {
-    margin-top: 0;
-    margin-bottom: 0;
+  
+    padding-left: var(--nb-container-spacings);
+    padding-right: var(--nb-container-spacings);
+    
+    &:not(:only-child) {
+      @include block-spacing;
```

Comment:
A supernova item inside a block that has the collection header hidden still has a margin-top because some default css styling and not being applied our default spacing rules.